setup.py

```diff
@@ -42,7 +42,9 @@
     ],
     entry_points = {
         "console_scripts": [
-            "embed = embed.__main__:main",
+            "pathogen-embed = pathogen_embed.__main__:main",
+            "pathogen-distance = pathogen_distance.__main__:main",
+            "pathogen-cluster = pathogen_cluster.__main__:main"
```

Comment:
Since this interface allows us to connect specific command line scripts to specific functions in a given `__main__` module, we could place all of the code for these different commands in the same module under different function names and refer to those here. For example:

```python
            "pathogen-embed = pathogen_embed.__main__:embed",
            "pathogen-distance = pathogen_embed.__main__:distance",
            "pathogen-cluster = pathogen_embed.__main__:cluster"
```

I think this approach would allow us to reference shared code from different functions without duplicating that code across multiple modules.